### PR TITLE
PFW-1298: Use PF-build-env 1.0.7

### PR DESCRIPTION
**Pull request by 3d-gussner**
Dependencies:

- [x] Update `PF-build.sh` 
- [x] Remove [travis trigger commit](https://github.com/prusa3d/Prusa-Firmware/pull/3422/commits/c37811c5a6518d7170dcd5844d65f3c8d503efde)
- [x] Change url to [prusa3d](https://github.com/prusa3d/Prusa-Firmware/pull/3422/commits/ea6ec785535467b1bc94ea93a0937d53138b3e3c#diff-4d2a8eefdf2a9783512a35da4dc7676a66404b6f3826a8af9aad038722da6823R12-R13)
     - [x]  https://github.com/prusa3d/PF-build-env/pull/5 has been merged
        - [x] https://github.com/prusa3d/Arduino_Boards/pull/16 has been merged

Changes:
- Based on Arduino IDE 1.8.19
- with [Arduino_boards 1.0.5](https://github.com/prusa3d/Arduino_Boards/tree/devel)

Resource savings:
|Variant| before flash/mem | after flash/mem | diff flash/mem |
|---------|-------------------------|----------------------|---------------------|
|MK3S | [245842/5655](https://app.travis-ci.com/github/prusa3d/Prusa-Firmware/builds/246764483#L305-L306) | [244808//5639](https://app.travis-ci.com/github/prusa3d/Prusa-Firmware/builds/246995135#L305-L306)| -1034/-16|
|MK3 | [246998/5686](https://app.travis-ci.com/github/prusa3d/Prusa-Firmware/builds/246764483#L481-L482) | [245560//5668](https://app.travis-ci.com/github/prusa3d/Prusa-Firmware/builds/246995135#L480-L481)| -1438/-18|
|MK25S | [218348/5530](https://app.travis-ci.com/github/prusa3d/Prusa-Firmware/builds/246764483#L657-L658) | [217296//5500](https://app.travis-ci.com/github/prusa3d/Prusa-Firmware/builds/246995135#L655-L656)| -1052/-30|

### CODE REVIEW

**Comment by 3d-gussner** [comment]
@mkbel Please ignore the `Configuration.h` commit as it was only to trigger travis.
Also this is a "prepare" PR for the final with url links to my fork for testing it.
As soon we get https://github.com/prusa3d/Arduino_Boards/pull/16 merged I will create a PR from my https://github.com/3d-gussner/PF-build-env-1/tree/v1.0.7 and create a release on prusa3d like my https://github.com/3d-gussner/PF-build-env-1/releases/tag/1.0.7-Linux64

File: build.sh [hunk 1]
```diff
@@ -1,18 +1,17 @@
...
 
-if [ ! -f "PF-build-env-Linux64-$BUILD_ENV.zip" ]; then
-    #wget https://github.com/3d-gussner/PF-build-env-1/releases/download/$BUILD_ENV-Linux64/PF-build-env-Linux64-$BUILD_ENV.zip || exit 3
-	wget https://github.com/prusa3d/PF-build-env/releases/download/$BUILD_ENV-Linux64/PF-build-env-Linux64-$BUILD_ENV.zip || exit 3
-fi
-
 if [ ! -d "../../PF-build-env-$BUILD_ENV" ]; then
+    if [ ! -f "PF-build-env-Linux64-$BUILD_ENV.zip" ]; then
+        wget https://github.com/3d-gussner/PF-build-env-1/releases/download/$BUILD_ENV-Linux64/PF-build-env-Linux64-$BUILD_ENV.zip || exit 3
+        #wget https://github.com/prusa3d/PF-build-env/releases/download/$BUILD_ENV-Linux64/PF-build-env-Linux64-$BUILD_ENV.zip || exit 3
```
Comment (mkbel) [inline build.sh]: Why it is not new build-env got from prusa3d repository?

Comment (3d-gussner) [reply build.sh]: The URLs will be changed to prusa3d repos and branches as soon the Arduino_boards and PF-build-env are approved and merged. 
I am using my forks and branches for testing.

**Review by DRracer — APPROVED** [review]

**Comment by 3d-gussner** [comment]
@mkbel URL has been updated.

**Review by leptun — APPROVED** [review]
Tested the default build options. The correct build env was used. The travis build uses the correct code. PF-build.sh uses the correct code.
All that remains is for the build server to be updated.

**Comment by 3d-gussner** [comment]
Server build has been updated in the morning. 🎉

**Pull request merged**